Sources/OpenTelemetrySdk/Trace/TraceConstants.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+public enum TraceConstants: String {
+    
+    /// Exception event conventions
+
+    /// An exception event must be named "exception"
+    case exception = "exception";
+    /// The type of the exception (its fully-qualified class name, if applicable). The dynamic type of the exception should be preferred over the static type in languages that support it.
+    case exceptionType = "exception.type";
+    /// SHOULD be set to true if the exception event is recorded at a point where it is known that the exception is escaping the scope of the span.
+    case exceptionEscaped = "exception.escaped";
+    /// The exception message.
```

Comment:
I suggest to note whether it should be localized or not (if this is possible in Swift to take a neutral - `en` version of a string).

Reply:
I think that depends on open-telemetry API spec. Should semantic conventions be localized? If so we should create a Issue to review all the semantic conventions to update localization.

Reply:
I've opened a ticket #115 for this.